README.md

```diff
@@ -1,2 +1,14 @@
-# labs
-An experimental public repository for holding various Optimizely Labs
+# <img src="./img/beaker.png" width="35px"> Optimizely Labs
+
+Optimizely Labs is a collection of tutorials ("labs") for working with Optimizely data and developer tools.
+
+Each lab contains a **focused** combination of documentation and code demonstrating how to solve a specific problem with Optimizely data or developer tools.  For example, [this lab](labs/query_enriched_event_data_with_spark) contains instructions for loading and querying [Optimizely Enriched Event data](https://docs.developers.optimizely.com/web/docs/enriched-events-export) using [Apache Spark](https://spark.apache.org/).
+
+Labs are **self-contained**, so all of the instructions and resources needed to run executable code can be found in the lab directory.  They're also **open-sourced** under the [Apache 2.0 license](http://www.apache.org/licenses/LICENSE-2.0), so you're free to modify and use them to solve problems in your own way.
```

Comment:
FYI, @byronopt suggested that some labs "link out" to content that already exists. For example, if Contentful has a great post on integrating Optimizley Full Stack with Contentful, we may prefer to have a lab that links directly to those instructions rather than reproducing them on our side.

I think this wording is fine since we haven't yet added those types of labs yet, and maybe the wording still applies, but something to keep in mind.

Reply:
I think that makes sense.  By "self-contained" I mean that everything you need to be able to run executable code should be in the lab directory.  Labs that link elsewhere could still follow this principle, right?